tests/README.md

```diff
@@ -224,22 +224,39 @@ Following services need to be supported:
 - `oscap` (`openscap-scanner` - the container has to be able to scan itself)
 - You may want to include another packages, as base images tend to be bare-bone and tests may require more packages to be present.
 
-Using Podman:
+#### Using Podman
 
 NOTE: With Podman, you have to run all the operations as root. Podman supports rootless containers, but the test suite internally uses a container exposing a TCP port. As of Podman version 0.12.1.2, port bindings are not yet supported by rootless containers.
 
+The root user therefore needs an key without passphrase, so it can log in to the container without any additional interaction.
```

Comment:
Is the SSH key a hard requirement or usability improvement?

Reply:
It is a requirement, because one usually doesn't know the root password. Moreover, one would have to enter it 3 times per scenario, which is IMO totally beyond usability.

Reply:
So it is a requirement, but also with usability effects.
What is confusing to me is that the sentence justifies the requirement with usability argument.